tests/unit/testplan/testing/test_pytest.py

```diff
@@ -96,7 +102,7 @@ def test_run_testcases_iter_testcase(pytest_test_inst):
     """Test running a single testcase iteratively."""
     all_results = list(
         pytest_test_inst.run_testcases_iter(
-            testsuite_pattern="pytest_tests.py::TestPytestBasics",
+            testsuite_pattern="examples/PyTest/pytest_tests.py::TestPytestBasics",
```

Comment:
What is the impact of this change? Can you show me the test_plan_top.py report please?

Reply:
http://testplan-db.ms.com/testplan/15bf8a587ff549faaf321f28a3b3c2e9
here is an example

Reply:
the pytest name and testsuite name duplicates, is there any way to refine this?

Reply:
The new fragment needs to be renamed to conform with latest convension.

Reply:
fragment updated. the pytest name is the start string of test suite name. for now I don't have a good idea for their names.